spec/index.bs

```diff
@@ -11763,9 +11763,8 @@ partial interface GPUDevice {
 
 ## Telemetry ## {#telemetry}
 
-User agents **may** dispatch an <dfn event for="GPUDevice">uncapturederror</dfn> event on a
-{{GPUDevice}} when a {{GPUError}} is generated that is not observed by any [=GPU error scope=]. The
-event must be of type {{GPUUncapturedErrorEvent}}.
+User agents **may** [=fire an event=] named <dfn event for="GPUDevice">uncapturederror</dfn> using {{GPUUncapturedErrorEvent}} on a
+{{GPUDevice}} when a {{GPUError}} is generated that is not observed by any [=GPU error scope=].
```

Comment:
Slight flipping of the sentence for clarity (cause -> consequence instead of consequence <- cause):

```suggestion
When a {{GPUError}} is generated that is not observed by any [=GPU error scope=], the user agent **may** [=fire an event=] named <dfn event for="GPUDevice">uncapturederror</dfn> at a {{GPUDevice}} using {{GPUUncapturedErrorEvent}}.
```